guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title:  "Continuous Delivery to Kubernetes on Azure"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab, you will create a Source to Prod pipeline targeting a Kubernetes cluster in Azure. In the end you will have a Jenkins instance with a sample job that builds a container image and pushes it to an Azure Container Registry. You will also have a Spinnaker instance with a sample pipeline that deploys the container image to a staging environment, waits for manual judgement, and then deploys the image to a production environment.
+
+## Setup
+
+Fork [this repository](https://github.com/azure-devops/spin-kub-demo) with a simple 'Hello World' application written in Go. You will commit a change to this fork in order to trigger the Source to Prod pipeline.
+
+## Deploy Dev Ops VM
+
+Run the following command to deploy the 'Continuous Delivery to Kubernetes' [Quickstart Template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-acr-spinnaker-k8s) in Azure. It will make sure you have the [Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) installed and interactively ask if you want to change any of the default behavior. **Important:** When asked, make sure to specify your fork of the sample git repository so that you can trigger the pipeline with your own commits.
+
+```bash
+bash <(curl -sL https://aka.ms/DeployDevOps)
+```
+
+The script should list several outputs after it has finished. Run the 'SSH' output to connect to your Jenkins and Spinnaker instances hosted on the same VM.
```

Comment:
I'd add a sentence explaining what the SSH command does (what ports will be forwarded)